tests/e2e-pw/utils/navigation/navigation.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { PlaywrightTestArgs } from '@playwright/test';
+import { BlockData } from '@woocommerce/e2e-types';
+
+/**
+ * Closes any modals in the editor if they are open.
+ */
+export const closeModalIfExists = async (
+	page: PlaywrightTestArgs[ 'page' ]
+) => {
+	// The modal close button can have different aria-labels, depending on the version of Gutenberg/WP.
+	// Newer versions (WP >=6.2) use `Close`, while older versions (WP <6.1) use `Close dialog`.
+	const closeButton = await page.$(
+		'.components-modal__header [aria-label="Close"], .components-modal__header [aria-label="Close dialog"]'
```

Comment:
I wonder if we can use 
```css
'.components-modal__header [aria-label^="Close"]'
```
or
```css
'.components-modal__header [aria-label*="Close"]'
```
instead of 
```css
'.components-modal__header [aria-label="Close"], .components-modal__header [aria-label="Close dialog"]'
```

https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors states:
> [attr^=value]
> Represents elements with an attribute name of attr whose value is prefixed (preceded) by value.
>
> [attr*=value]
> Represents elements with an attribute name of attr whose value contains at least one occurrence of value within the string.

`[attr^=value]` can be used to check is a string starts with something, while `[attr*=value]` can be used to check if the string contains something. In this particular case, I'd say that `[attr^=value]` is more suitable. What are your thoughts on this?

Reply:
Thanks for the suggestion and for writing up the ^ and * selectors.

In this case I think I would prefer to leave it as is. In my opinion it is better to be explicit about what we're looking for instead of using wildcards.

Reply:
Being explicit sounds good, @opr. Just wanted to point out an alternative. That said, with your comment, I think being explicit is indeed the better approach here.